ci/test/00_setup_env_native_valgrind.sh

```diff
@@ -5,11 +5,11 @@
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
 export LC_ALL=C.UTF-8
-
+export TEST_RUNNER_EXTRA="--exclude feature_block,feature_abortnode,wallet_createwallet"
```

Comment:
```suggestion
export TEST_RUNNER_EXTRA="--exclude feature_block,rpc_bind.py --ipv4,feature_abortnode,wallet_createwallet"
```

Due to failure here: https://travis-ci.org/bitcoin/bitcoin/jobs/632228754#L2879